### PR TITLE
Process get_info without a session established

### PR DESCRIPTION
**Pull request by jamoralp**
This will allow to implement a ping feature between a client and an agent, without the need of establishing a XRCE session.

### CODE REVIEW

**Comment by richiware** [comment]
Build status:
* Linux [Build Status](http://jenkins.eprosima.com:8080/job/Micro-XRCE-DDS-Agent%20Manual%20Linux/295/)
* Windows [Build Status](http://jenkins.eprosima.com:8080/job/Micro-XRCE-DDS-Agent%20Manual%20Windows/294/)

File: src/cpp/processor/Processor.cpp [hunk 1]
```diff
@@ -47,6 +47,25 @@ void Processor<EndPoint>::process_input_packet(
...
+                    process_get_info_packet(std::move(input_packet), output_packet);
+                    server_.push_output_packet(std::move(output_packet));
+                    break;
+                }
+                default:
+                {
+                    break;
+                }
+            }
             if (input_packet.message->get_subheader().submessage_id() == dds::xrce::CREATE_CLIENT)
```
Comment (pablogs9) [inline src/cpp/processor/Processor.cpp:69]: This is done in the line above right? you are calling twice to `process_create_client_submessage`

Comment (jamoralp) [reply src/cpp/processor/Processor.cpp]: Fixed :man_facepalming:

**Comment by richiware** [comment]
Build status:
* Linux [Build Status](http://jenkins.eprosima.com:8080/job/Micro-XRCE-DDS-Agent%20Manual%20Linux/298/)
* Windows [Build Status](http://jenkins.eprosima.com:8080/job/Micro-XRCE-DDS-Agent%20Manual%20Windows/297/)

**Review by pablogs9 — DISMISSED** [review]

**Comment by richiware** [comment]
Build status:
* Linux [Build Status](http://jenkins.eprosima.com:8080/job/Micro-XRCE-DDS-Agent%20Manual%20Linux/305/)
* Windows [Build Status](http://jenkins.eprosima.com:8080/job/Micro-XRCE-DDS-Agent%20Manual%20Windows/304/)

**Pull request merged**